README.md

```diff
@@ -139,10 +139,17 @@ The configuration and cache are saved to ~/.config/fisherman and ~/.cache/fisher
 
 The fishfile and plugins are saved to ~/.config/fish by default.
 
-To customize this location:
+To customize this location, add the following to your ~/.config/fish/config.fish file:
 
 ```fish
-set -U fish_path ~/my/path
+set -g fish_path ~/my/path
```

Comment:
@greenie Why change the scope from -U to -g?

Reply:
Ahh, looks like I'd copy-and-pasted the whole thing from my own fish config. I have -g instead of -U there as I felt that having fish_path written to ~/.config/fish/fishd.xxx every time I started fish seemed excessive. Happy to change it back to -U if I'm mistaken and that's a better thing to do.

Reply:
True! I hadn't considered that. Changed it back to -U :)